src/components/cards/TierCard/TierCard.tsx

```diff
@@ -1,14 +1,16 @@
+import { useRegistrationState } from '../../../state/contexts/RegistrationState';
 import { TierCardProps } from '../../../types';
 import { TIER_DATA } from '../../../utils/constants';
 import { CircleCheck } from '../../icons';
 import './styles.css';
 
-function TierCard({ tier, setTier, selectedTier }: TierCardProps) {
+function TierCard({ currentTier }: TierCardProps) {
```

Comment:
nit: `selectedTier`

Reply:
Its not really selected - the selected tier is "tier" from the useRegistrationState. currentTier is more like.... this.tierData. maybe tierNumber / tierName would be more apt?